source/user/pbc/scf.rst

```diff
@@ -0,0 +1,186 @@
+  :cite:`Sundararaman2013`, which is more appropriate than spherical truncation
+  for anisotropic cells.  The finite-size error of the exchange energy decays as
+  :math:`\exp(-aN_k^{1/3})`.  Only supported with plane-wave density fitting
+  (FFTDF).
+
+An example calculation with exchange treated with the spherically truncated
+Coulomb potential is shown here::
+
+    kmf = scf.KRHF(cell, kpts=kpts, exxdiv='vcut_sph').run()
+
+Band structure calculations
+===========================
+
+After an SCF calculation has been performed, the band structure can be
+calculated non-self-consistently along a k-point path using the
+``SCF.get_bands(kpts)`` function, where ``kpts`` is a list of k-points along
+which the band structure is desired.
+
+.. warning::
+   For Hartree-Fock or hybrid DFT, the discontinuity of the exchange potential
+   at :math:`G=0` is problematic for band structure calculations.  Using
+   :attr:`exxdiv='vcut_sph'` with FFTDF is recommended instead.  Alternatively,
+   the SCF procedure can be repeated at each k-point, which is much more
+   expensive but allows the use of any :attr:`exxdiv` or density fitting.
+
+See :source:`examples/pbc/09-band_ase.py` for an example DFT band structure
+calculation.
+
+Add-ons
+=======
+
+All molecular SCF add-ons are also available for periodic SCF but
+must be accessed through the molecular :mod:`pyscf.scf.addons` module.  
+Here we highlight a few of the most useful add-ons.
+
+Linear dependencies
+-------------------
+
+The dense packing of atoms in solids combined with the use of diffuse
+atom-centered basis functions is responsible for frequent linear dependencies.
+The linear dependency can be eliminated by Cholesky orthogonalization::
+
+    from pyscf import scf as mol_scf
+    kmf = scf.KRHF(cell, kpts=kpts)
+    kmf = mol_scf.addons.remove_linear_dep_(kmf).run()
+
+Smearing
+--------
+
+For metals or small band gap semiconductors, it can be useful to smear the
+orbital occupation numbers away from integer values.  This can improve SCF
+convergence and can expedite convergence to the thermodynamic limit with k-point
+sampling.  Because this approach assumes a finite electronic temperature, it
+yields an entropy and free energy::
+
+    kmf = scf.KRHF(cell, kpts=kpts)
+    kmf = scf.addons.smearing_(kmf, sigma=0.01, method='fermi').run()
+    print('Entropy = %s' % kmf.entropy)
+    print('Free energy = %s' % kmf.e_free)
+    print('Zero temperature energy = %s' % ((kmf.e_tot+kmf.e_free)/2))
+
+Fermi-Dirac smearing (:attr:`method='fermi'`) and Gaussian smearing
+(:attr:`method='gauss'`) are supported.
+
+.. warning::
+   Because most functions in PySCF assume integer occupations, they may fail if
+   combined with a mean-field calculation that was performed with smearing.
+
+Stability
+---------
+
+Periodic SCF solutions can be checked with stability analysis::
+
+    kmf = scf.KRHF(cell).run()
+    kmf.stability()
```

Comment:
Is smearing supported in combination of stability analysis?

Reply:
Not supported